cdap-docs/developer-manual/source/advanced/configuring-resources.rst

```diff
@@ -111,13 +111,13 @@ in the ``default`` namespace, and would like to set the memory used by its YARN
     .. tabbed-parsed-literal::
        :tabs: "CDAP CLI"
 
-       |cdap >| set preferences flow 'flowlet.collector.system.resources.memory=1024' PurchaseHistory.PurchaseFlow
+       |cdap >| set preferences flow 'flowlet.collector.system.resources.memory=1024' MyApp.PurchaseFlow
 
   or by using a ``curl`` call:
 
     .. tabbed-parsed-literal::
 
-      $ curl -w"\n" -X PUT "http://example.com:11015/v3/namespaces/default/apps/PurchaseHistory/flows/PurchaseFlow/preferences" \
+      $ curl -w"\n" -X PUT "http://example.com:11015/v3/namespaces/default/apps/MyApp/flows/PurchaseFlow/preferences" \
```

Comment:
should this section be removed entirely? There are no more flows.

Reply:
removed.